arbnode/batch_poster.go

```diff
@@ -154,24 +154,25 @@ type BatchPosterConfig struct {
-	L1BlockBound                   string                      `koanf:"l1-block-bound" reload:"hot"`
-	L1BlockBoundBypass             time.Duration               `koanf:"l1-block-bound-bypass" reload:"hot"`
-	UseAccessLists                 bool                        `koanf:"use-access-lists" reload:"hot"`
-	GasEstimateBaseFeeMultipleBips arbmath.UBips               `koanf:"gas-estimate-base-fee-multiple-bips"`
-	Dangerous                      BatchPosterDangerousConfig  `koanf:"dangerous"`
-	ReorgResistanceMargin          time.Duration               `koanf:"reorg-resistance-margin" reload:"hot"`
-	CheckBatchCorrectness          bool                        `koanf:"check-batch-correctness"`
+	ErrorDelay                                          time.Duration               `koanf:"error-delay" reload:"hot"`
+	CompressionLevel                                    int                         `koanf:"compression-level" reload:"hot"`
+	DASRetentionPeriod                                  time.Duration               `koanf:"das-retention-period" reload:"hot"`
+	GasRefunderAddress                                  string                      `koanf:"gas-refunder-address" reload:"hot"`
+	DataPoster                                          dataposter.DataPosterConfig `koanf:"data-poster" reload:"hot"`
+	RedisUrl                                            string                      `koanf:"redis-url"`
+	RedisLock                                           redislock.SimpleCfg         `koanf:"redis-lock" reload:"hot"`
+	ExtraBatchGas                                       uint64                      `koanf:"extra-batch-gas" reload:"hot"`
+	Post4844Blobs                                       bool                        `koanf:"post-4844-blobs" reload:"hot"`
+	IgnoreBlobPrice                                     bool                        `koanf:"ignore-blob-price" reload:"hot"`
+	ParentChainWallet                                   genericconf.WalletConfig    `koanf:"parent-chain-wallet"`
+	L1BlockBound                                        string                      `koanf:"l1-block-bound" reload:"hot"`
+	L1BlockBoundBypass                                  time.Duration               `koanf:"l1-block-bound-bypass" reload:"hot"`
+	UseAccessLists                                      bool                        `koanf:"use-access-lists" reload:"hot"`
+	GasEstimateBaseFeeMultipleBips                      arbmath.UBips               `koanf:"gas-estimate-base-fee-multiple-bips"`
+	Dangerous                                           BatchPosterDangerousConfig  `koanf:"dangerous"`
+	ReorgResistanceMargin                               time.Duration               `koanf:"reorg-resistance-margin" reload:"hot"`
+	CheckBatchCorrectness                               bool                        `koanf:"check-batch-correctness"`
+	PeriodToAllowPostingBatchWithOnlyBatchPostingReport time.Duration               `koanf:"period-to-allow-posting-batch-with-only-batch-posting-report"`
```

Comment:
Need to find a simpler name.

A batch posting report is automatically added (by the inbox smart-contract) every time a batch is posted. Orbit deployers don't usually understand too well what is a "batch-posting-report". They will experience it as maximum delay between batches even if they are empty.

I want something close to "max-deay" that shows the difference.

Maybe "max-empty-batch-delay"?